packages/core-utils/src/itinerary.js

```diff
@@ -118,7 +118,7 @@ export function getMapColor(mode) {
   mode = mode || this.get("mode");
   if (mode === "WALK") return "#444";
   if (mode === "BICYCLE") return "#0073e5";
-  if (mode === "SUBWAY") return "#f00";
+  if (mode === "SUBWAY") return "#e60000"; /* from #f00 to comply with WCAG */
```

Comment:
We probably don't need a comment about the previous value here.

Reply:
Removed in 1f02f4c26f30f25588f4519499764a86dcf4bfc5